mantidproject/mantid
# Add figures to reflectometry release notes

Adds a couple of figures to Reflectometry release notes.

**To test:**

Build reflectometry release notes and check that the new figures are displayed correctly.

Fixes #18772 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
